apps/api/R/auth.R

```diff
@@ -29,15 +29,11 @@ get_crypt_pass <- function(username, password, secretkey = NULL) {
 #* @author Tezan Sahu
 validate_crypt_pass <- function(username, crypt_pass) {
 
-  dbcon <- PEcAn.DB::betyConnect()
```

Comment:
Not sure why the dbcon was dropped from this function, as it's needed to query the users table. If it's already open, then we should probably pass it in as an argument rather than depend on it as a global variable.

Reply:
Addressed in 526db40fa.